en_ewt-ud-dev.conllu

```diff
@@ -8012,7 +8012,7 @@
 2	did	do	AUX	VBD	Mood=Ind|Tense=Past|VerbForm=Fin	4	aux	4:aux	SpaceAfter=No
 3	n't	not	PART	RB	_	4	advmod	4:advmod	_
 4	realize	realize	VERB	VB	VerbForm=Inf	0	root	0:root	_
-5	how	how	ADV	WRB	PronType=Int	6	advmod	6:advmod	_
+5	how	how	SCONJ	WRB	PronType=Int	6	mark	6:mark	_
```

Comment:
What's the proper treatment of *how much/little/often/...* N when serving as a verb complement?

Reply:
(Also: *I don't know __how much longer it will take__*)

Reply:
I don't think it's the clausal mark here: "how" modifies "much" (compare "very much", the answer to "how much")